src/main/java/com/zendesk/maxwell/scripting/WrappedRowMap.java

```diff
@@ -0,0 +1,77 @@
+package com.zendesk.maxwell.scripting;
+
+import com.zendesk.maxwell.replication.BinlogPosition;
+import com.zendesk.maxwell.row.RowMap;
+
+import java.util.LinkedHashMap;
+
+// we pass this little wrapper into the javascript interface.
+// this class is here so that we're not exposing the full RowMap,
+// and so we can snake_case things properly.
+public class WrappedRowMap {
+	private final RowMap row;
+
+	public WrappedRowMap(RowMap row) {
+		this.row = row;
+	}
+
+	public LinkedHashMap<String, Object> getData() {
+		return row.getData();
+	}
+
+	public LinkedHashMap<String, Object> getOld_data() {
+		return row.getOldData();
+	}
+
+	public LinkedHashMap<String, Object> getExtra_attributes() {
+		return row.getExtraAttributes();
+	}
+
+	public String getType() {
+		return row.getRowType();
+	}
+
+	public String getTable() {
+		return row.getTable();
+	}
+
+	public String getDatabase() {
+		return row.getDatabase();
+	}
+
+	public String getPosition() {
+		BinlogPosition p = row.getPosition().getBinlogPosition();
```

Comment:
NPE if row.getPosition() will ever be NULL.  Not sure if that can ever happen though.